# service for gaze control

---
bool done
time goal_stamp
